lib/gopher.c

```diff
@@ -83,7 +83,7 @@ static CURLcode gopher_do(struct connectdata *conn, bool *done)
   char *sel;
   char *sel_org = NULL;
   ssize_t amount, k;
-  int len;
+  size_t len;
```

Comment:
Practically unlikely, but using `int` could result in negative values or overflows.